upgrade_test.py

```diff
@@ -362,6 +363,7 @@ def _upgrade_node(self, node, upgrade_sstables=True, new_scylla_repo=None, new_v
         InfoEvent(message='upgrade_node - starting to "_update_argus_upgraded_version"').publish()
         self._update_argus_upgraded_version(node, new_ver)
         InfoEvent(message='upgrade_node - ended to "_update_argus_upgraded_version"').publish()
+        enable_default_filters(sct_config=self.params)
```

Comment:
just calling it won't cut it,

you need to be able to call it with multiple version ? i.e. once for base, and once for target version

those are not update/changed in SctConfiguration.

it's need to go down to the SkipPerIssue implementation to be aware also what is target version